library/std/src/sys/windows/fs.rs

```diff
@@ -57,6 +57,9 @@ pub struct DirEntry {
     data: c::WIN32_FIND_DATAW,
 }
 
+unsafe impl Send for OpenOptions {}
+unsafe impl Sync for OpenOptions {}
+
```

Comment:
There was some argument as to whether this is a serious Robustness To Refactoring regression. If so, I can replace this with a wrapper type for just `c::LPSECURITY_ATTRIBUTES` -- I don't know if such a wrapper already exists.

Reply:
> Robustness To Refactoring

I searched for this term on Google and there are a total of *two* results, none of which have any relation to the term itself. What does this mean?

Reply:
It's not a specific term, just Capitalization for Emphasis. (i.e. this is a potential regression to correctness in the face of refactoring in the future, such as the addition of another `!Send`/`!Sync` field in the future.)